packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -78,6 +78,18 @@ export interface ICluster extends IResource, ec2.IConnectable {
    */
   readonly clusterEncryptionConfigKeyArn: string;
 
+  /**
+   * An Open ID Connect Issuer Url
+   * @attribute
+   */
+  readonly openIdConnectIssuerUrl: string;
+
+  /**
+   * An Open ID Connect Provider
+   * @attribute
```

Comment:
This isn't really an attribute in that it doesn't represent a one to one mapping of some cluster runtime data.

```suggestion

```